src/runtimes/go/index.ts

```diff
@@ -139,13 +140,18 @@ const zipFunction: ZipFunction = async function ({
     const zipPath = `${destPath}.zip`
     const zipOptions = {
       destPath: zipPath,
-      filename: basename(destPath),
+      filename: featureFlags.zisi_golang_use_al2 ? 'bootstrap' : basename(destPath),
       runtime,
     }
 
     await zipBinary({ ...zipOptions, srcPath: binary.path, stat: binary.stat })
 
-    return { config, path: zipPath, entryFilename: zipOptions.filename }
+    return {
+      config,
+      path: zipPath,
+      entryFilename: zipOptions.filename,
+      goUseAL2Runtime: featureFlags.zisi_golang_use_al2,
```

Comment:
What will we do with this property?

Reply:
We'll read it in open-api, so we can propagate it through nf-server into BitBalloon and functions-origin. They can then read it and use `provided.al2` as the AWS runtime, if it's present. If you have an idea for a solution where we don't need to propagate this so far, i'd be happy to hear it!

Reply:
Could we use the existing `runtimeVersion` property? https://github.com/netlify/zip-it-and-ship-it/blob/626d63d4de399693d5f1cbd329e95c6b62651166/src/manifest.ts#L16

Reply:
I guess so, but that'd still require changes to all three services. But that field is probably better-suited. I'll make that update!

Reply:
done in https://github.com/netlify/zip-it-and-ship-it/pull/1559/commits/80a3c66ba1ddb49d6867b0de7debdc1eddab4ddb